espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.java

```diff
@@ -34,22 +32,24 @@
  *
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
-@RunWith(AndroidJUnit4.class)
 @LargeTest
 public class EspressoServerRunnerTest {
     private final Server espressoServer = Server.getInstance();
 
     @Test
     public void startEspressoServer() throws InterruptedException, IOException, DuplicateRouteException {
-        try {
-            espressoServer.start();
-
-            while (!espressoServer.isStopRequestReceived()) {
-                Thread.sleep(1000);
+        System.out.println(System.getProperty("UNITTEST"));
```

Comment:
This is not a good thing to put println into a production code

Reply:
That was  a mistake.